guides/md/training_keras_models_on_cloud.md

```diff
@@ -0,0 +1,381 @@
+# Training Keras models with TensorFlow Cloud
+
+**Author:** [Jonah Kohn](https://jonahkohn.com)<br>
+**Date created:** 2020/08/11<br>
+**Last modified:** 2020/08/11<br>
+**Description:** In-depth usage guide for TensorFlow Cloud.
+
+
+<img class="k-inline-icon" src="https://colab.research.google.com/img/colab_favicon.ico"/> [**View in Colab**](https://colab.research.google.com/github/keras-team/keras-io/blob/master/guidesipynb/training_keras_models_on_cloud.ipynb)  <span class="k-dot">•</span><img class="k-inline-icon" src="https://github.com/favicon.ico"/> [**GitHub source**](https://github.com/keras-team/keras-io/blob/master/guidestraining_keras_models_on_cloud.py)
+
+
+
+---
+## Introduction
+[TensorFlow Cloud](https://github.com/tensorflow/cloud) is a Python package that
+provides APIs for a seamless transition from local debugging to distributed training
+in Google Cloud. It simplifies the process of training TensorFlow models on the
+cloud into a single, simple function call, requiring minimal setup and no changes
+to your model. TensorFlow Cloud handles cloud-specific tasks such as creating VM
+instances and distribution strategies for your models automatically. This guide
+will demonstrate how to interface with Google Cloud through TensorFlow Cloud,
+and the wide range of functionality provided within TensorFlow Cloud. We'll start
+with the simplest use-case.
+
+---
+## Setup
+We'll get started by installing TensorFlow Cloud, and importing the packages we
+will need in this guide.
+
+
+```python
```

Comment:
Not sure why this shows up as a Python code block -- maybe something we'd have to fix in the generation script. WDYT?

Reply:
I noticed that too, I think it's because the markdown is somehow connected to the notebook. In a notebook code cell, it would make sense to execute this as python.